lib/resValidator.js

```diff
@@ -130,3 +139,15 @@ function findOAPathMatchingPathname(pathname, OAPaths) {
   });
   return OAPath;
 }
+
+function getServerUrlUsed(openApiFullPath, servers) {
+  if (!servers) {
+    return null;
+  }
+  const serverUsed = servers.find(server => openApiFullPath.startsWith(server.url));
+  return serverUsed ? serverUsed.url : null;
```

Comment:
nit: would `return serverUsed.url || null` work here?

Reply:
btw how come we're using `null` instead of `undefined`? (I don't know which is best)

Reply:
to be honest if it's falsey then it'll be null anyway so perhaps we don't need that line at all, just return the result of the find function.

Reply:
Unless for clarity you think it's better to make it explicit that it could return null?

Reply:
I now remember why I did this. If no server is found (i.e. `serverUsed = null`) then trying to access `serverUsed.url` would throw an exeption.